src/build_rpm_from_dir

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+srcdir=$1; shift
+name=$1; shift
+outdir=$1; shift
+
+sha256=$(cd "${srcdir}" && find . ! -type d -print0 | xargs -0 -r sha256sum \
+            | sha256sum | cut -f 1 -d ' ')
+
+rpm="$name-$sha256-1.noarch.rpm"
```

Comment:
this could be confusing because the rpm might show up as `downgraded` just based on the checksum, right? Or does `$name` include something like `fedora-coreos-29.1` in it?

Reply:
Hmm good point, I think we could add a simple counter for this. Or just make `Version` be the counter and `Release` include the SHA256. We can't use the OSTree version itself since that's generated by `rpm-ostree compose tree` which takes this RPM as input. :recycle:

Reply:
I would just make version be a timestamp and the release be the sha256. Can you make the sha256 the first 7 characters of the sha256?

Reply:
This is done now!